scripts/delete_tickets_2012.sh

```diff
@@ -1,4 +1,4 @@
-for raw_id in `cat data/y2012_tickets`
+for raw_id in `cat y2012_tickets`
```

Comment:
Looks like these files are still in the `data` subdirectory?

Reply:
ok, going to correct this and the others so that data/ is used in both creating the files and consuming them.